### PR TITLE
Minutes for 2024-02-14 CG meeting

### PR DESCRIPTION
**Pull request by hzbarcea**

### CODE REVIEW

File: meetings/2024-02-14.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+* eP: PROPOSAL: attach label and ping Maxime to resolve it
+* HZ: +1
+
+### Recommend predicate for a literal media type (Content-Type)
+URL: https://github.com/solid/specification/issues/537
+
+* RG: IETF will standardize format for content types. Maybe we should register ???
+* PAC: ... subtypes, i agree that having an easy way to reference media type could be useful.
+* AC: should this fall under the notion of specifying container description?
+* eP: I'm using it in Description Resource, if app wants to show an icon for attachment etc.
```
Comment (TallTed) [inline meetings/2024-02-14.md:139]:
```suggestion
* eP: I'm using it in Description Resource, if app wants to show an icon for attachment, etc.
```

**Review by TallTed — CHANGES_REQUESTED** [review]
minor cleanup. note `???` block wants further fill-in.

File: meetings/2024-02-14.md [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+### Scribes
+* elf Pavlik
+
+
+### Introductions
+* name: text
+
+### Status of the WG
+* PAC: I'm having conversation with ??? The next step is to submit a new charter. The horizontal review should be relatively fast. Then we will have votes of AC, if we still have formal objections we should go ASAP to the new council. Many objections has been addressed, some objections can be argued in front of the council.
+* HZ: I recall that only one objection Sarven was considering valid. It was related to the chair selection. There was also something about the scope and the explainer. Tim has alredy written an explainer. What exactly has to be rewritten.
```
[the remark this answers is not in the document]
Comment (elf-pavlik) [reply meetings/2024-02-14.md:44]: I don't think the name of the specific person is essential, so unless  @pchampin wants to fill it in, I would just go ahead and merge it with the ???

**Review by VirginiaBalseiro — APPROVED** [review]
Please squash when merging. Also, you can batch commit suggestions and add them all in one commit from the Files changed view instead of making individual commits.

**Pull request merged**